partner_contact_in_several_companies/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Contacts in several partners",
+    "summary": "Allow to have one contact in several partners",
+    "version": "12.0.2.0.0",
```

Comment:
@foliveira-odoogap That's make me think you forget to embed migration commit.

Reply:
My bad, I'll correct that as soon as possible!

Reply:
@rousseldenis I embed the migration commit.